algebird-core/src/main/scala/com/twitter/algebird/Eventually.scala

```diff
@@ -162,7 +162,7 @@ class EventuallyRing[E, O](convert: O => E)(mustConvert: O => Boolean)(implicit
 ) extends EventuallyGroup[E, O](convert)(mustConvert)
     with Ring[Either[E, O]] {
 
-  override def one = Right(Ring.one[O])
+  override def one = Right(Ring.one[O]) // scalafix:ok
```

Comment:
same comment as above.

Reply:
ahh sorry i  probably explained myself badly ... yeah we can make it `override def one: Right[E, O] = Right(Ring.one[O])` but not `override def one: Either[E, O] = Right(Ring.one[O])`. I felt it was to tight ... but on a second thought maybe it's ok for these.

Reply:
actually, I think promising the zero/one are Right is actually good (and true).